tools/ColonCatcher/2017-08-09-colon_operator_log.txt

```diff
@@ -0,0 +1,53 @@
+..code\controllers\subsystem\shuttle.dm Lines: 402 Total Colons: 1
+..code\datums\ai_laws.dm Lines: 237 Total Colons: 1
+..code\datums\browser.dm Lines: 134, 135, 138, 141 Total Colons: 6
+..code\datums\datumvars.dm Lines: 342 Total Colons: 1
+..code\datums\mind.dm Lines: 805, 806, 816, 821 Total Colons: 5
+..code\datums\riding.dm Lines: 45 Total Colons: 1
+..code\datums\antagonists\datum_clockcult.dm Lines: 68, 69 Total Colons: 2
+..code\game\communications.dm Lines: 295 Total Colons: 3
+..code\game\gamemodes\objective.dm Lines: 847 Total Colons: 1
+..code\game\machinery\newscaster.dm Lines: 364, 415, 428, 478, 944 Total Colons: 5
+..code\game\machinery\overview.dm Lines: 99, 245 Total Colons: 2
+..code\game\machinery\robot_fabricator.dm Lines: 21, 23, 24, 25, 28 Total Colons: 5
+..code\game\machinery\status_display.dm Lines: 153 Total Colons: 4
+..code\game\machinery\computer\communications.dm Lines: 504, 635 Total Colons: 2
+..code\game\machinery\computer\medical.dm Lines: 62, 67 Total Colons: 2
+..code\game\machinery\computer\security.dm Lines: 121, 128, 209, 229, 385, 401 Total Colons: 6
+..code\game\machinery\doors\brigdoors.dm Lines: 193 Total Colons: 4
+..code\game\mecha\mecha.dm Lines: 717 Total Colons: 1

```

Comment:
https://github.com/tgstation/tgstation/blob/master/code/game/mecha/mecha.dm#L717

Also highlighting this one actually, as it's a multi line ternary (DUN DUN DUN!!!1!!)